frontend/src/app/features/team-planner/team-planner/planner/team-planner.component.html

```diff
@@ -16,6 +16,7 @@
       *ngIf="calendarOptions"
       [options]="calendarOptions"
       class="op-team-planner--calendar"
+      [ngClass]="{'op-team-planner--calendar_empty' : !(principalIds$ | async)?.length && !(showAddAssignee$ | async)}"
```

Comment:
Could you add a separate field with a pipe'd observable for that? I think more than one async in a condition is really confusing